spec/observables/dom/ajax-spec.ts

```diff
@@ -558,26 +555,17 @@ describe('ajax', () => {
       async: false,
     };
 
-    ajax(obj).subscribe(
-      (x: any) => {
-        expect(x.status).to.equal(200);
-        expect(x.xhr.method).to.equal('GET');
-        expect(x.xhr.async).to.equal(false);
-        expect(x.xhr.timeout).to.be.undefined;
-        expect(x.xhr.responseType).to.equal('');
-      },
-      () => {
-        throw 'should not have been called';
-      }
-    );
+    ajax(obj).subscribe();
 
-    const request = MockXMLHttpRequest.mostRecent;
+    const mockXHR = MockXMLHttpRequest.mostRecent;
 
-    expect(request.url).to.equal('/flibbertyJibbet');
+    expect(mockXHR.url).to.equal('/flibbertyJibbet');
+    // Open was called with async `false`.
+    expect(mockXHR.async).to.be.false;
```

Comment:
Here's the most important part of this test. This is only set when the XHR's `open` method is called, and it defaults to `true`.